recipe/meta.yaml

```
@@ -65,8 +65,8 @@ requirements:
     - qtpy >=1.5.0
     - rtree >=0.9.7
     - setuptools >=49.6.0
-    - sphinx >=0.6.6
-    - spyder-kernels >=2.2.0,<2.3.0
+    - sphinx <4.4.0,>=0.6.6
```

```suggestion
    - sphinx >=0.6.6,<4.4.0
```

Why is this restriction necessary?

---

Seems like Sphinx requires importlib_metadata >=4.4.0 for python < 3.10 starting in Sphinx 4.4.0 while flake8 requires importlib_metadata <4.3 for python <3.8